packages/editor/src/components/parts/web-service/Exception.tsx

```diff
@@ -0,0 +1,41 @@
+import { IVY_SCRIPT_TYPES } from '@axonivy/inscription-protocol';
+import { useWebServiceData } from './useWebServiceData';
+import { PathCollapsible, PathFieldset } from '../common';
+import { Checkbox, ScriptInput, useFieldset } from '../../widgets';
+
+export const Exception = () => {
+  const { config, defaultConfig, updateException } = useWebServiceData();
+
+  const conditionFieldset = useFieldset();
+  const messageFieldset = useFieldset();
+
+  return (
+    <PathCollapsible
+      path='exception'
+      label='Exception'
+      defaultOpen={
+        config.exception.enabled !== defaultConfig.exception.enabled ||
+        config.exception.condition !== defaultConfig.exception.condition ||
+        config.exception.message !== defaultConfig.exception.message
+      }

```

Comment:
as nothing else is in `config.exception` (at least I think this is the case right?), it is totally fine to compare the whole object:
`deepEqual(config.exception, defaultConfig.exception)`